pom.xml

```diff
@@ -778,7 +778,7 @@
                     <plugin>
                         <groupId>org.owasp</groupId>
                         <artifactId>dependency-check-maven</artifactId>
-                        <version>1.2.9</version>
+                        <version>1.4.5</version>
```

Comment:
I did look at them. I had added the `skipTestScope` to the configuration but somehow that was lost in my rebasing. I will add that option back in.